app/controllers/prison/visits_controller.rb

```diff
@@ -72,4 +80,30 @@ def visit_page(visit)
       prison_deprecated_visit_path(visit)
     end
   end
+
+  def load_visit
```

Comment:
I fixed the bad merge, I missed the BookingResponseContext. Let's pair on the presenter idea